include/mbedtls/ssl_ciphersuites.h

```diff
@@ -31,14 +31,27 @@
 extern "C" {
 #endif
 
+#if defined(MBEDTLS_DEPRECATED_WARNING)
+
+#define MBEDTLS_DEPRECATED __attribute__((deprecated))
+MBEDTLS_DEPRECATED typedef int deprecated_ciphersuite_id_t;
```

Comment:
Please keep `typedef` in column 0 (and the type being defined last before the semicolon), it helps readers.

Reply:
Done.